modules/utxo-lib/src/bitgo/zcash/ZcashPsbt.ts

```diff
@@ -128,6 +124,9 @@ export class ZcashPsbt extends UtxoPsbt<ZcashTransaction<bigint>> {
     this.tx.consensusBranchId = getDefaultConsensusBranchIdForVersion(network, version);
   }
 
+  // For Zcash transactions, we do not have to have non-witness utxo data for non-segwit
+  // transactions because zcash hashes the value directly and therefore does not need to
+  // check the previous transaction.
```

Comment:
Realized [this comment](https://github.com/BitGo/BitGoJS/pull/3210/files#r1087261135) after merging the other PR, adding the comment here